picker-starter/src/picker.config.ts

```diff
@@ -0,0 +1,40 @@
+import { getProductFilters, queryCategories, queryProducts } from '@/data'
+import { defineConfig } from '@/core'
+import { StoryblokIcon } from './components'
+import { optionsSchema } from './options-schema'
+
+export default defineConfig((options) => {
+  return {
+    title: 'Picker Starter',
+    icon: StoryblokIcon,
+    options: {
+      example: {
+        limit: '10',
+        selectOnly: 'product',
+      },
+      validate: () => {
+        const validationResult = optionsSchema.validate(options, {
```

Comment:
I'm going to get rid of the Joi here and perform raw validations, without using any external libraries.